requirements.txt

```diff
@@ -1,5 +1,6 @@
 APScheduler==3.5.1
 boto==2.48.0
+boto3==1.5.20
```

Comment:
Do we need explicit pins for botocore or any other dependencies of boto3?

Reply:
Not sure about this. Reviewing the changelog (https://github.com/boto/boto3/blob/develop/CHANGELOG.rst) it looks like the manage the dependencies pretty meticulously. Opinions?

Reply:
There are only 3 dependencies and boto3 specifies a maximum version for each of them, so I guess I'm okay with leaving them unpinned if we want. My gut says leaving things unpinned always leads to surprises though.